src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/SQLFunctions.java

```diff
@@ -334,7 +334,7 @@ private static String quoteParams(String... params) {
 
     private static Tuple<String, String> dateFunctionTemplate(String methodName, SQLExpr field) {
         String name = randomize(methodName);
-        return new Tuple<>(name, def(name, doc(field) + ".date." + methodName));
+        return new Tuple<>(name, def(name, doc(field) + ".value." + methodName));
```

Comment:
What is this change for?

Reply:
This is the breaking changes in ES 7.0. [getDate() and getDates removed](https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#_getdate_and_getdates_removed).

Reply:
cool, good to know!